src/TodoListItem.jsx

```diff
@@ -1,7 +1,7 @@
 import item from "./TodoList";
 
 
-function TodoListItem ({todo}) {
+function TodoListItem ({ todo }) {
   return  (
     <li key={todo.id}>
```

Comment:
It's not necessary to add the key keyword since li it's only used along with the map method (in the parent component)